<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootApplyWindow</class>
 <widget class="QWidget" name="RootApplyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/rootapplywindow_icon.png</normaloff>:/icons/rootapplywindow_icon.png</iconset>
  </property>
  <widget class="QPushButton" name="visibleIconPushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

border: 20px;

};
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resource/icons/visible.png</normaloff>resource/icons/visible.png</iconset>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>10</y>
     <width>401</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QPushButton" name="rootApplyPushButton">
      <property name="text">
       <string>Підтвердити</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="rootLoginLabel">
        <property name="text">
         <string>root-login</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Показати пароль</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="rootPasswordLabel">
        <property name="text">
         <string>root-password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="rootPasswordLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="rootLoginLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="hiddenIconPushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

border: 20px;

};
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resource/icons/hidden.png</normaloff>resource/icons/hidden.png</iconset>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>hiddenIconPushButton</zorder>
  <zorder>visibleIconPushButton</zorder>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
